Cabal/doc/buildinfo-fields-reference.rst

```diff
@@ -180,15 +180,15 @@ asm-sources
 autogen-includes
     * Monoidal field
     * Available since ``cabal-version: 3.0``.
-    * Documentation of :pkg-field:`autogen-includes`
+    * Documentation of :pkg-field:`custom-setup:autogen-includes`
```

Comment:
this file is auto-generated. I can fix the generator to do the right thing.

Yet, `autogen-includes` is definitely **not** `custom-setup` field.

Reply:
Cool. I was having a hard time finding where those fields come from. I was basically looking for `#pkg-config-SOMETHING` in the generated htmls :smile: